resources/prosody-plugins/mod_token_verification.lua

```diff
@@ -33,6 +33,13 @@ log("debug",
 	tostring(host), tostring(token_util.appId), tostring(token_util.appSecret),
 	tostring(token_util.allowEmptyToken));
 
+-- option to disable room modification (sending muc config form) for guest that do not provide token
+local disable_room_moderation_on_empty_token;
+local function load_config()
+    disable_room_moderation_on_empty_token = module:get_option_boolean("disable_room_moderation_on_empty_token");
```

Comment:
Is there a default value? Can we prefix these with the module name so we know what they affect? E.g. token_verification_require_token_for_moderation?

Reply:
Will rename it. Default is nil which evaluates as false...